cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetectorTest.java

```diff
@@ -255,7 +278,8 @@ private void testFixAnomaly(AnomalyType anomalyType)
     EasyMock.expect(mockKafkaCruiseControl.getLoadMonitorTaskRunnerState())
             .andReturn(LoadMonitorTaskRunner.LoadMonitorTaskRunnerState.RUNNING).times(1, 4);
 
-    startPeriodicDetectors(mockDetectorScheduler, mockGoalViolationDetector, mockMetricAnomalyDetector, mockDiskFailureDetector, executorService);
+    startPeriodicDetectors(mockDetectorScheduler, mockGoalViolationDetector, mockMetricAnomalyDetector, mockDiskFailureDetector,
```

Comment:
This function seems to be quite verbose -- can we move some of the logic to functions to make what it is testing more clear?

Reply:
It would be hard since most of the code is to create mock objects, if we move these code to another method, the return value(we need to return a lot of mock object) will be hard in Java.